### ISSUE TITLE
Breaks my tests

### ISSUE THREAD

**Issue by jorisw** [pocka/storybook-addon-designs#48]
Hi,

`storybook-addon-designs` is breaking my tests, using jest:

```
 FAIL  src/tests/storyshots.test.js
  ● Test suite failed to run

    ./node_modules/storybook-addon-designs/lib/index.js:1
    ({"Object.<anonymous>":function(module,exports,require,__dirname,__filename,global,jest){import addons, { makeDecorator } from '@storybook/addons';
                                                                                             ^^^^^^

    SyntaxError: Cannot use import statement outside a module

      1 | import React from 'react';
      2 | import { action } from '@storybook/addon-actions';
    > 3 | import { withDesign } from 'storybook-addon-designs';
```

I've tried both versions `5.2.0` and `5.3.0-alpha.1`.

If I remove the module and comment out the `withDesign` related lines, all my tests complete without errors.

I have tried various suggestions in https://github.com/facebook/jest/issues/9395 , to no avail.

**Comment by pocka** [comment]
This addon is intended to be used by Storybook. So you should only include the addon in your story files, not tests.

BTW, I noticed the `main` field directs the transpiled file that uses ES Modules. I'll fix it by adding CJS files then placing it to the `main` field and by adding a `module` field that directs the current `lib/index.js`.

**Comment by pocka** [comment]
fixed at v5.2.1
